### PR TITLE
cb added for inviting (switchContext value)

### PR DESCRIPTION
**Pull request by imashok02**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -206,7 +206,8 @@ var FacebookInstant = Class(function () {
           },
           text: opts.message,
           image: opts.image,
-          template: 'invite'
+          template: 'invite',
+          cb: opts.cb
```
Comment (nasimafeef) [inline js/index.js]: why???

Comment (imashok02) [reply js/index.js]: Modified with some changes

File: js/index.js [hunk 2]
```diff
@@ -195,20 +195,9 @@ var FacebookInstant = Class(function () {
...
-            name: player.getName(),
-            action: opts.action
-          },
-          text: opts.message,
-          image: opts.image,
-          template: 'invite'
-        })
-      }))
+      .then(function () {
+        opts.cb();
```
Comment (nasimafeef) [inline js/index.js]: then is not needed. you can keep it inside chooseAsync's then.

File: js/index.js [hunk 3]
```diff
@@ -195,22 +195,10 @@ var FacebookInstant = Class(function () {
...
-          },
-          text: opts.message,
-          image: opts.image,
-          template: opts.template || 'invite'
-        });
-
+      .then(function () {
+        opts.cb();
         fbInstant.logEvent('invite_sent');
-      }))
```
Comment (rampr) [inline js/index.js:213]: trailing whitespace

**Review by nasimafeef — APPROVED** [review]

**Pull request merged**